src/Administration/Resources/app/administration/src/module/sw-settings-shipping/component/sw-settings-shipping-price-matrix/index.js

```diff
@@ -498,5 +499,9 @@ export default {
         updateShowAllPrices() {
             this.showAllPrices = true;
         },
+
+        changeNextPrice(index) {
+            this.priceGroup.prices[index + 1].quantityStart = this.priceGroup.prices[index].quantityEnd;
```

Comment:
What happens if `prices[index].quantityEnd` also exceeds `prices[index + 2].quantityStart`? Or if `prices[index].quantityEnd >prices[index + 1].quantityStart`?

Reply:
Currently nothing. But it's even nothing without the changes.

I could change it to bump them all up, but that might be annoying for 2 reasons:

1. If we change it to bump it up, the current values will be changed. So it removes what was there, which might be annoying.
2. For some bizarre reason, that is super annoying, when you try to change a value that is already in the DB, it resets to that value on the first keypress. Really annoying behaviour, but it means that even if we change it, the moment you are going to try change it again, it's going to reset to the db value on first keypress.

Reply:
All right, then leave them. Why were the values out of sync with the database? What failed? Does this behaviour only occur for the next price in the price group or for all prices? => Should we reload the whole entity to avoid disconnection?